bokeh/document/locking.py

```diff
@@ -84,7 +84,7 @@ def without_document_lock(func: F) -> NoLockCallback[F]:
 
     '''
     @wraps(func)
-    def _wrapper(*args: Any, **kw: Any):
+    def _wrapper(*args: Any, **kw: Any) -> Any:
         return func(*args, **kw)
```

Comment:
I actually think the return should possibly be None here

Reply:
Possibly, there a *a lot* of places where `return` statements are used unnecessarily.